core/shared/src/main/scala/scalgorand/domain/accounts/Address.scala

```diff
@@ -0,0 +1,20 @@
+package scalgorand.domain.accounts
+
+import zio.json.{DeriveJsonDecoder, DeriveJsonEncoder, JsonDecoder, JsonEncoder}
+
+case class Address(
+  LEN_BYTES: Int,
+  bytes: Array[Byte],
```

Comment:
is this in the return json?

Reply:
The address is returned as a string in the json response.
I will remove the class in the next PR.